installer/templates/phx_ecto/data_case.ex

```diff
@@ -44,10 +44,11 @@ defmodule <%= app_module %>.DataCase do
 
   """
   def errors_on(changeset, field) do
-    for {message, opts} <- Keyword.get_values(changeset.errors, field) do
+    Ecto.Changeset.traverse_errors(changeset, fn {message, opts} ->
       Enum.reduce(opts, message, fn {key, value}, acc ->
         String.replace(acc, "%{#{key}}", to_string(value))
       end)
-    end
+    end)
+    |> Map.get(field, [])
```

Comment:
Credo has a check that does not allow a single pipe. So we should rather pipeline the whole way, starting with changeset or break it apart using a variable.